pandas/io/tests/test_excel.py

```diff
@@ -401,6 +401,25 @@ def test_read_excel_blank_with_header(self):
         actual = self.get_exceldf('blank_with_header', 'Sheet1')
         tm.assert_frame_equal(actual, expected)
 
+    # GH 12292 : error when read one empty column from excel file
+    def test_read_one_blank_col(self):
+        df = pd.DataFrame(
+            [["", 1, 100],
+             ["", 2, 200],
+             ["", 3, 300],
+             ["", 4, 400]]
+        )
+        with ensure_clean(self.ext) as path:
+            df.to_excel(path, 'one_blank_col', index=False, header=False)
+            actual = read_excel(
+                path,
+                'one_blank_col',
+                parse_cols=[0],
+                header=None
+            )
+        expected = DataFrame()
+        tm.assert_frame_equal(actual, expected)
```

Comment:
can you add another test with headers? (but blank values), not sure if that would even trigger the error, but covering bases